website/docs/quickstart.md

```diff
@@ -79,6 +79,20 @@ Suppose your app is to be named `my_app`, then execute:
 flutter_rust_bridge_codegen create my_app
 ```
 
+
+
+Suppose your app is named `my_app` and specify organization to `com.example`, then your appid is `com.example.my_app`
```

Comment:
Hmm, I am not sure whether it is great to put into `quickstart`, since quickstart needs to be brief in order to be "quick". But anyway, no worries about it, I will merge and then tweak a bit!

Reply:
i see. thanks.